.github/workflows/ci.yml

```diff
@@ -10,7 +10,7 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          - 1.6 # LTS
+          - lts
```

Comment:
We can't require `lts` in project.toml, right? So I think an explicit version in CI might be better.

Reply:
That's a good point. Ideally, I guess one would want to test the oldest supported version and LTS - the first could be hardcoded (since one has to update the Project.toml manually anyway when changing it) and the second achieved with the `lts` option. But it's unnecessarily many tests if both coincide...